admin-dev/themes/default/template/layout-ajax.tpl

```diff
@@ -79,13 +79,13 @@
 		<div class="alert alert-warning">
 			<button type="button" class="close" data-dismiss="alert">&times;</button>
 			{if count($warnings) > 1}
-				{l s='There are %d warnings.' sprintf=count($warnings)}
+				{l s='There are %d warnings.' sprintf=count($warnings) d='Admin.Notifications.Error'}
 				<span style="margin-left:20px;" id="labelSeeMore">
-					<a id="linkSeeMore" href="#" style="text-decoration:underline">{l s='Click here to see more'}</a>
-					<a id="linkHide" href="#" style="text-decoration:underline;display:none">{l s='Hide warning'}</a>
+					<a id="linkSeeMore" href="#" style="text-decoration:underline">{l s='Click here to see more' d='Admin.Actions'}</a>
```

Comment:
```suggestion
					<a id="linkSeeMore" href="#" style="text-decoration:underline">{l s='See more' d='Admin.Actions'}</a>
```

Reply:
You really don't want to change this wording? ;-)

Reply:
Fixed in https://github.com/PrestaShop/PrestaShop/pull/15111/commits/863895811b2dbaf8b1f5e3002491e73b01418dd3 😄